src/main/java/taxi/Main.java

```diff
@@ -10,25 +14,79 @@
     public static void main(String[] args) {
         ManufacturerService manufacturerService = (ManufacturerService)
                 injector.getInstance(ManufacturerService.class);
+        DriverService driverService = (DriverService)
+                injector.getInstance(DriverService.class);
+        CarService carService = (CarService)
+                injector.getInstance(CarService.class);
 
         Manufacturer toyota = new Manufacturer("Toyota", "Japan, Toyota City");
         Manufacturer honda = new Manufacturer("Honda", "Japan");
         Manufacturer koenigsegg = new Manufacturer("Koenigsegg", "Sweden, Angelholm");
+        Manufacturer nissan = new Manufacturer("Nissan", "Japan");
+        Manufacturer bmw = new Manufacturer("BMW", "Germany");
+
+        Driver one = new Driver("001", "0001");
+        Driver two = new Driver("002", "0002");
+        Driver three = new Driver("003", "0003");
+        Driver four = new Driver("004", "0005");
+        Driver five = new Driver("005", "0005");
+
+        Car yaris = new Car("Yaris GR", toyota);
+        Car markII = new Car("MarkII", toyota);
+        Car civic = new Car("Civic type r", honda);
+        Car nsx = new Car("NSX", honda);
+        Car gemera  = new Car("Gemera", koenigsegg);
+        Car skyline = new Car("Skyline R35", nissan);
+
 
         manufacturerService.create(toyota);
         manufacturerService.create(honda);
         manufacturerService.create(koenigsegg);
+        manufacturerService.create(nissan);
+        manufacturerService.create(bmw);
 
         System.out.println(manufacturerService.getAll());
 
-        manufacturerService.delete(3L);
-
-        System.out.println(manufacturerService.getAll());
+        manufacturerService.delete(5L);
 
         Manufacturer manufacturerUpdate = manufacturerService.get(2L);
-        manufacturerUpdate.setCountry("Japan, Minato City");
+        manufacturerUpdate.setCountry("Japan, Minato-City");
 
         System.out.println(manufacturerService.getAll());
 
+        driverService.create(one);
+        driverService.create(two);
+        driverService.create(three);
+        driverService.create(four);
+        driverService.create(five);
+
+        Driver driverUpdate = driverService.update(two);
+        driverUpdate.setLicenceNumber("39800");
+        System.out.println(driverService.get(2L));
```

Comment:
```suggestion
        System.out.println(driverService.update(driverUpdate));
```
let's update the driver after we altered some information about him
same for manufacturer